lib/settings.coffee

```diff
@@ -1,8 +1,15 @@
 module.exports =
   config:
+
+    fontSize:
+      title: 'Font Size'
+      description: 'Change the UI font size. Needs to be between 10 and 20.'
+      type: 'string'
```

Comment:
@lee-dohm Is there a way to use an `integer` but have its default be `default: undefined` or `default: ''`. When I try that I get an error and it only works when adding an actual number, like `default: 12`.

The reason for having an undefined default is so that when you don't enter a number, the Layout Mode will adjust the font size.